Implements v3.0.0 (#20)
roukmoute/hashids-bundle: README.md

    # if set to true, it will continue with the next available param converters
    passthrough:     false

>     # if set to true, it try to convert all arguments in controller

Reviewer: ```suggestion
    # if set to true, it tries to convert all arguments passed to the controller
```

Author: Done

    auto_convert:    false
```

## Usage

```php

use Hashids\HashidsInterface;

public function postShow(HashidsInterface $hashids): Response
{
    $hashids->…
}
```

> Next it's the same things of [official documentation](http://hashids.org/php/).

Reviewer 2: should use `https`

Author: Done

}
```

> You could have several hashids one the same URL with prefix `_hash_`.

Reviewer: ```suggestion
You could have several hashids in the same URL prefixed with  `_hash_`.
```

Author: Done

```php
/**
 * @Route("/users/{_hash_user}/status/{_hash_status}")
 */
public function getAction(int $user, int $status)
{
}
```

The keys must be the same as in parameters controller:

```php
/**
 *                       _hash_user _hash_status
 *                              ↕            ↕

> public function getAction(int $user, int $status)

Reviewer: ```suggestion
 * public function getAction(int $user, int $status)
```

Author: Done
